internal/encoding/encodingtest/hex.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Finobo
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package encodingtest
+
+import (
+	"encoding/hex"
```

Comment:
This should use the `internal/encoding` package

Reply:
Hi @developerfred this is not resolved, you need to use `internal/encoding` package not `encoding/hex` package. Then select the relevant method.